travis-ci/managers/fuzzbuzz.sh

```diff
@@ -16,7 +16,13 @@ git clone https://github.com/google/oss-fuzz /tmp/oss-fuzz
 cd /tmp/oss-fuzz
 sudo ./infra/helper.py pull_images
 
+# docker doesn't like colons in filenames so let's create a directory
+# whose name can be consumed by the -v option.
+# https://github.com/google/oss-fuzz/issues/2428
+t=$(mktemp -d)
+sudo mount --bind "$REPO_ROOT" "$t"
```

Comment:
Hmm, my first instinct was to ask for a trap to unmount this and remove the directory. But even before this change, the script couldn't be run more than once, so this wouldn't make any difference. It's ok as is then.

Reply:
Thank you for the review! I agree it would be great to make the script idempotent :-) In the meantime, I'm trying to figure out why `meson` mysteriously fails with
```sh
+ meson --werror -Db_sanitize=address,undefined -Dsplit-usr=true build
The Meson build system
Version: 0.50.1
Source dir: /home/vsts/agents/2.150.3/d:\a/1/s
Build dir: /home/vsts/agents/2.150.3/d:\a/1/s/build
Build type: native build
Project name: systemd
Project version: 242
Native C compiler: cc (gcc 5.4.0 "cc (Ubuntu 5.4.0-6ubuntu1~16.04.11) 5.4.0 20160609")
Build machine cpu family: x86_64
Build machine cpu: x86_64

meson.build:46:0: ERROR: Program or command '/home/vsts/agents/2.150.3/d:/a/1/s/tools/add-git-hook.sh' not found or not executable

A full log can be found at /home/vsts/agents/2.150.3/d:\a/1/s/build/meson-logs/meson-log.txt
##[error]Bash exited with code '1'.
```
from time to time. Initially I thought it had something to do with a colon but I can't seem to reproduce it locally.

I'll go ahead and merge the PR because it helps to fix the issue with helper.py and the script succeeded on Azure Pipelines (though apparently if code is pushed two times in a row too fast the status vanishes so it's necessary to run it manually there).

Reply:
`meson` doesn't like backslashes apparently:
```
git clone --depth=1 https://github.com/systemd/systemd test\\test
cd test\\test/
meson ./build
The Meson build system
Version: 0.50.1
Source dir: /home/vagrant/test\test
Build dir: /home/vagrant/test\test/build
Build type: native build
Project name: systemd
Project version: 242
Native C compiler: cc (gcc 8.3.1 "cc (GCC) 8.3.1 20190223 (Red Hat 8.3.1-2)")
Build machine cpu family: x86_64
Build machine cpu: x86_64

meson.build:46:0: ERROR: Program or command '/home/vagrant/test/test/tools/add-git-hook.sh' not found or not executable

A full log can be found at /home/vagrant/test\test/build/meson-logs/meson-log.txt
```

Reply:
I think they fixed some of those issues, but apparently not all. Please file a meson bug.

Reply:
I opened https://github.com/mesonbuild/meson/issues/5400 almost as soon as I discovered it :-) Though it seems to be a variation of https://github.com/mesonbuild/meson/issues/1564 (which is unlikely to be fixed).

In principle the same bind-mount trick could help to get rid of backslashes to make `meson` happy but I kind of hope it was a glitch of some kind on Azure Pipelines and it will never happen again.